source/loader/layers/validation/ur_valddi.cpp

```diff
@@ -3246,20 +3247,29 @@ urProgramCreate(
         if (NULL == phProgram) {
```

Comment:
Curious why `NULL` is used over `nullptr`?

Reply:
For this `NULL == phProgram` is automatically generated, I'm not sure where the source of it is. For the new checks I decided to match the existing usage of `NULL` to make the spec consisten.

I beleive the reason to use `NULL` is that the API defined in `ur_api.h` is a C API where `nullptr` is not a valid keyword.

Reply:
Good to know, thanks!